pipelinerl/run_actor.py

```diff
@@ -338,15 +338,7 @@ def __init__(
             self.rollout_processes.append(process)
 
     def init_stats(self):
-        self.reward_stats = make_stats_dict()
-        self.step_stats = make_stats_dict()
-        self.no_errors_stats = make_stats_dict()
-        self.no_answer_stats = make_stats_dict()
-        self.success_stats = make_stats_dict()
-        self.prompt_tokens = make_stats_dict()
-        self.output_tokens = make_stats_dict()
-        self.num_turns = make_stats_dict()
```

Comment:
I think `self.num_turns` and `self.overflows` will be gone with this PR. We need these metrics!!

Reply:
@AlexPiche